DigitalLearningSolutions.Web/Helpers/SelectListHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.AspNetCore.Mvc.Rendering;
+
+    public static class SelectListHelper
+    {
+        public static IEnumerable<SelectListItem> GetOptionsWithSelectedText(List<(int id, string name)> options, string? selectedName)
```

Comment:
I wonder if this should be named "MapToSelectListItems" to make it clearer what this function does. 🤔 And `selectedName` could be `selectedOptionName ` for extra clarity (I wasn't sure what the "selected name" was initially)